catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchSource.java

```diff
@@ -811,6 +860,11 @@ public void setDistanceTolerance(int distanceTolerance) {
     }
   }
 
+  public void setPollInterval(Integer interval) {
+    this.pollInterval = interval;
```

Comment:
The metatype file says this value has a minimum value of 1. Should we be enforcing that here?

Reply:
good catch